crates/agglayer-node/src/node.rs

```diff
@@ -243,6 +247,16 @@ impl Node {
             .await?;
 
         info!("Certificate orchestrator started.");
+
+        let debug_handle = AdminAgglayerImpl::new(
```

Comment:
Nit: `debug_handle` name no longer matches the type name but it may not be worth it updating all the variable names, not sure.